packages/slider/README.md

```diff
@@ -1,64 +1,124 @@
+### Continuous
 
-          ```npm i @webcomponents/webcomponentsjs```
+<img src="images/basic.png" width="140px">
 
-      - Add webcomponents polyfills to your HTML page
+```html
+<script type="module">
+  import '@material/mwc-slider';
+</script>
 
-          ```<script src="@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>```
+<mwc-slider value="25" min="10" max="50"></mwc-slider>
+```
 
-  1. Add mwc-slider to your project:
+### Discrete
 
-      ```npm i @material/mwc-slider```
+<img src="images/discrete.gif" width="163px">
 
-  1. Import the mwc-slider definition into your HTML page:
+```html
+<mwc-slider
+    step="5"
+    pin
+    markers
+    max="50"
+    value="10">
+</mwc-slider>
+```
 
-      ```<script type="module" src="@material/mwc-slider/index.js"></script>```
+### Styled
 
-      Or into your module script:
+<img src="images/styled.gif" width="188px">
 
-      ```import {Slider} from "@material/mwc-slider"```
+```html
+<style>
+  mwc-slider {
+    --mdc-theme-secondary: orange;
+    --mdc-theme-text-primary-on-dark: green;
+  }
+</style>
+<mwc-slider
+    pin
+    markers
+    max="50"
+    value="10"
+    step="5">
+</mwc-slider>
+```
 
-  1. Create an instance of mwc-slider in your HTML page, or via any framework that [supports rendering Custom Elements](https://custom-elements-everywhere.com/):
+## API
 
-      ```<mwc-slider></mwc-slider>```
 
-  1. Install the Polymer CLI:
+### Properties/Attributes
 
-      ```npm i -g polymer-cli```
+| Name      | Type      | Default | Description
+| --------- | --------- |-------- | -----------
+| `value`   | `number`  | `0`     | Current value of the slider.
+| `min`     | `number`  | `0`     | Minimum value of the slider.
+| `max`     | `number`  | `100`   | Maximum value of the slider.
+| `step`    | `number`  | `0`     | Sets the step of the slider making it discrete. Set to 0 to make continuous. Note: if `step` is greater than 0 but less than 1, it will be rounded up to 1.
+| `pin`     | `bookean` | `false` | Shows the thumb pin on a discrete slider.
+| `markers` | `bookean` | `false` | Shows the marker on the slider's track of a discrete slider.
 
-  1. Run the development server and open a browser pointing to its URL:
+### Methods
 
-      ```polymer serve```
+| Name     | Description
+| -------- | -------------
+| `layout() => void` | Updates the internal size model. May be required if the slider is resized or repositioned horizontally.
 
-  > mwc-slider is published on [npm](https://www.npmjs.com/package/@material/mwc-slider) using JavaScript Modules.
-  This means it can take advantage of the standard native JavaScript module loader available in all current major browsers.
-  >
-  > However, since mwc-slider uses npm convention to reference dependencies by name, a light transform to rewrite specifiers to URLs is required to get it to run in the browser. The polymer-cli's development server `polymer serve` automatically handles this transform.
+### Listeners
```

Comment:
Is this section necessary? Doesn't strike me as the kind of thing 99% of users would need to see in the docs, they would just assume the slider responds to user events.